spec/models/spree/order_contents_spec.rb

```diff
@@ -139,6 +139,7 @@
       end
 
       it "updates the order's enterprise fees if completed" do
+        allow(order).to receive(:shipment).and_return(create(:shipment, order:))
```

Comment:
Why do we need this with the new Rails version? May it come from the new defaults you activated in the config?